docs/development.md

```diff
@@ -183,6 +191,154 @@ molecule verify -s name_of_your_test_case
+```bash
+cd /home/user/aurora/ansible/playbooks/molecule_ec2_teleop
+ANSIBLE_ROLES_PATH="/home/user/aurora/ansible/roles" molecule create -s teleop_server_check_desktop_icons_docker_ec2
+```
+This will create an ec2 instance, connect to it, and start running aurora. For us to connect to this instance, we will need it's IP address and key.
+
+The key will be generated and written to the following path:
+`/home/user/.cache/molecule/molecule_ec2_teleop/teleop_server_check_desktop_icons_docker_ec2/ssh_key`.
+
+The IP can either be found by scrolling back up to the `TASK [Wait for SSH] ************************************************************` section of the console printout, where you can find:
+
+```
+    "item": {
+        "address": "52.56.228.199",
+        "identity_file": "/home/user/.cache/molecule/molecule_ec2_teleop/teleop_server_check_desktop_icons_docker_ec2/ssh_key",
+        "instance": "teleop_server_check_desktop_icons_docker_ec2_",
+        "instance_ids": [
+            "i-0c2e66409c355f288"
+        ],
+        "port": 22,
+        "region": "eu-west-2",
+        "user": "ubuntu"
+    },
+```
+
+OR, by examining current connections in a new terminal (terminal 2) in the aurora_dev container:
+
+```bash
+sudo apt update && sudo apt install -y iproute2
+ss | grep ssh
+```
+(will return something like `tcp   ESTAB 0      0                 172.17.0.2:34430     52.56.228.199:ssh` if there is currently a connection between your machine and the ec2 instance)
+
+In terminal 2, we can now connect to the ec2 instance:
+```bash
+ssh -i /home/user/.cache/molecule/molecule_ec2_teleop/teleop_server_check_desktop_icons_docker_ec2/ssh_key ubuntu@52.56.228.199
+```
+
+In terminal 1, we can continue to run the steps in the [check](https://github.com/shadow-robot/aurora/blob/bdff8c474cc063877a26fdff5b4347b1a171add6/ansible/playbooks/molecule_ec2_teleop/molecule/teleop_server_check_desktop_icons_docker_ec2/molecule.yml#L60C1-L67C14) section in sequence. Eventually, when the test fails, being able to connect to the ec2 instance where the test is running can make it much easier to find out what's gone wrong and fix it.
+
+e.g - to run the next step in the `check` sequence:
+
+```bash
+cd /home/user/aurora/ansible/playbooks/molecule_ec2_teleop
+ANSIBLE_ROLES_PATH="/home/user/aurora/ansible/roles" molecule converge -s teleop_server_check_desktop_icons_docker_ec2
+```
+
+> [!CAUTION]
+> By running these stages individually (or, by interrupting the full `... molecule test` command), it is possible (easy) to never reach the `destroy` part of the check sequence. Doing this will leave an ec2 instance running forever, costing us money. It is STRONGLY ADVISED to log in to aws ec2 and see if any ec2 instances have been left alive. They will have a key name of `molecule_key_aurora...*`. Make sure no PR checks are running against aurora first, you don't want to delete an instance that is in use by someone else.
```

Comment:
presumably we could add a check for this somewhere

Reply:
(as in another job that runs periodically and checks for orphaned test instances)